straxen/plugins/event_processing.py

```diff
@@ -375,7 +375,7 @@ def compute(self, events):
             (170118_1327, pax_file('XENON1T_s2_xy_ly_SR1_v2.2.json'))]),
    strax.Option(
         'elife_file',
-        default=aux_repo + '3548132b55f81a43654dba5141366041e1daaf01/strax_files/elife.npy',
+        default=('elife_constant', 700000., True),
```

Comment:
According to the help the number should be a string.

Reply:
update the help accordingly